javascript/consumer.js

```diff
@@ -0,0 +1,28 @@
+import { createConsumer } from '@rails/actioncable'
+
+function isConsumer (object) {
+  if (object) {
+    try {
+      return (
+        object.constructor.name === 'Consumer' &&
+        object.connect &&
+        object.disconnect &&
+        object.send
+      )
+    } catch {}
```

Comment:
I know that ESsomething doesn't require catch to take an arg, but doesn't that mean this won't work unless ESsomething is present? I think you should just pass the arg.